dpctl/CMakeLists.txt

```diff
@@ -82,23 +83,25 @@ add_custom_target(_build_time_create_dpctl_include ALL
     COMMAND ${CMAKE_COMMAND} -E make_directory ${DPCTL_INCLUDE_DIR}/syclinterface
     COMMAND ${CMAKE_COMMAND} -E make_directory ${DPCTL_INCLUDE_DIR}/syclinterface/Support
     COMMAND ${CMAKE_COMMAND} -E make_directory ${DPCTL_INCLUDE_DIR}/syclinterface/Config
-    DEPENDS DPCTLSyclInterface
+    DEPENDS DPCTLSyclInterface DPCTLTensorInterface
```

Comment:
@oleksandr-pavlyk I want the headers copied before `DPCTLTensorInterface` target starts, how do I do that?